airbyte-config/config-persistence/src/main/java/io/airbyte/config/persistence/ConfigRepository.java

```diff
@@ -1627,8 +1631,7 @@ public Stream<ConnectorBuilderProject> getConnectorBuilderProjectsByWorkspace(fi
 
     return database
```

Comment:
should we add a @NotNull annotation for the input `workspaceId` to make sure it's not misused?